org.zdoom.Raze.yaml

```diff
@@ -0,0 +1,121 @@
+app-id: org.zdoom.Raze
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: "23.08"
+command: raze.sh
+
+finish-args:
+- --device=dri
+- --socket=wayland
+- --socket=fallback-x11
+- --share=ipc
+- --share=network
+- --socket=pulseaudio
+
+# Based on Raze
+# We redirect the original ~/.config/raze
+- --persist=.config/raze
+
+# Controller support on SteamDeck and such (copied from GZDoom)
+- --device=all
+
+# Simpler way of preventing KDE related errors
+- --env=KDE_FULL_SESSION=false
+
+cleanup:
+- /app/include
+- /app/lib/*.a
+- /app/lib/*.la
+- /app/lib/pkgconfig
+
+modules:
+
+- shared-modules/linux-audio/libinstpatch.json
+- shared-modules/linux-audio/fluidsynth2.json
+
+- name: libsndfile
+  buildsystem: autotools
+  sources:
+  - sha256: 3799ca9924d3125038880367bf1468e53a1b7e3686a934f098b7e1d286cdb80e
+    type: archive
+    url: https://github.com/libsndfile/libsndfile/releases/download/1.2.2/libsndfile-1.2.2.tar.xz
+    x-checker-data:
+      project-id: 13277
+      stable-only: true
+      type: anitya
+      url-template: https://github.com/libsndfile/libsndfile/releases/download/$version/libsndfile-$version.tar.xz
+
+
+- name: game-music-emu
+  buildsystem: cmake-ninja
+  config-opts:
+  - "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
+  sources:
+  - sha256: 626e8a104e0dadd10ef6519a67aca880c7b40f81471659f1935b61754e12fc7b
+    type: archive
+    url: https://bitbucket.org/mpyne/game-music-emu/downloads/game-music-emu-0.6.3.tar.gz
+    x-checker-data:
+      project-id: 866
+      stable-only: true
+      type: anitya
+      url-template: https://bitbucket.org/mpyne/game-music-emu/downloads/game-music-emu-$version.tar.gz
+
+- name: zmusic
+  buildsystem: cmake-ninja
+  config-opts:
+  - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+  sources:
+  - sha256: da818594b395aa9174561a36362332b0ab8e7906d2e556ec47669326e67613d4
+    type: archive
+    url: https://github.com/zdoom/ZMusic/archive/1.1.12.tar.gz
+    x-checker-data:
+      project-id: 153600
+      stable-only: true
+      type: anitya
+      url-template: https://github.com/zdoom/ZMusic/archive/$version.tar.gz
+
+- name: raze
+  buildsystem: cmake-ninja
+  config-opts:
+  - "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
+  - "-DSYSTEMINSTALL=1"
+  cleanup:
+  - "/lib/*.a"
+  - "/lib/*.la"
+  - "/lib/pkgconfig"
+  - "/include"
+  sources:
+  - type: git
+    url: https://github.com/ZDoom/Raze.git
+    tag: 1.9
+    commit: 9af265b5ea1ab2872368ae58ad3b5490fab658f1
+  post-install:
+  - install -Dm 644 soundfont/raze.sf2 /app/share/games/raze/soundfonts/raze.sf2
+
+- name: integration
```

Comment:
this doesn't need to be a separate module.

Reply:
Not quite sure how to go about this one, I've been pretty much following what GZDoom is doing for it's build, except for adding the whole build into "shared-modules" and only have the launcher built in the application yaml (as GZDoom does it), instead I have it defined as a separate module, as some earlier attempts at flatpaking the app suggested, would appreciate any advice.

Reply:
> Not quite sure how to go about this one, I've been pretty much following what GZDoom is doing for it's build,

Two wrong don't make a right.

Reply:
Sorry, didn't mean that as a justification, just that  being new to flatpak packaging I didn't know how to properly include the second step in the one in install, thank you very much for providing a suggestion.